suite-native/intl/src/en.ts

```diff
@@ -9,6 +9,35 @@ export const en = {
             title: 'My portfolio balance',
         },
     },
+    moduleConnectDevice: {
+        connectCrossroads: {
```

Comment:
is this screen? I think we should add screen suffix then

Reply:
https://github.com/trezor/trezor-suite/pull/9511/commits/e7b879fe00fb16146234f54fec1eb774fd06e3c2